app/assets/stylesheets/all/components/_forms.scss

```diff
@@ -19,6 +19,19 @@ form.form_form {
   }
 }
 
+// Override bootstrap on dl's and dt's for read only questioning
+dt {
```

Comment:
don't know best place to put this css stuff

Reply:
I think it's fine in here but I'd put a generic comment like "dl can be used for non bulleted lists of things in a read only form view. These styles put 20px of space between each dl, but not on either end."